esy.json

```diff
@@ -16,31 +16,35 @@
     "build_static": "dune build --profile=static --release"
   },
   "dependencies": {
-    "ocaml": "4.12.",
+    "@opam/alcotest": "1.5.0",
+    "@opam/cmdliner": "1.0.4",
+    "@opam/data-encoding": "*",
+    "@opam/digestif": "*",
+    "@opam/domainslib": "0.4.1",
+    "@opam/dream": "*",
     "@opam/dune": "*",
-    "@opam/ppx_deriving": "*",
-    "@opam/ppx_deriving_yojson": "*",
+    "@opam/hex": "*",
     "@opam/lwt": "*",
     "@opam/mirage-crypto": "*",
+    "@opam/mirage-crypto-ec": "*",
     "@opam/mirage-crypto-pk": "*",
     "@opam/mirage-crypto-rng": "*",
-    "@opam/mirage-crypto-ec": "*",
-    "@opam/dream": "*",
-    "@opam/piaf": "*",
     "@opam/mrmime": "0.3.2",
-    "@opam/hex": "*",
+    "@opam/piaf": "*",
+    "@opam/ppx_blob": "*",
+    "@opam/ppx_deriving": "*",
+    "@opam/ppx_deriving_yojson": "*",
+    "@opam/ppx_expect": "github:renatoalencar/ppx_expect:ppx_expect.opam#94d2862f6df315955de9951d8c77f47085a9a876",
```

Comment:
I don't know why esy is messing up the `esy.json` file, but in case anyone tries to fix it I need this dependency to have this resolution.